cargo-dist/src/errors.rs

```diff
@@ -303,11 +303,8 @@ pub enum DistError {
     },
 
     /// Linkage report can't be run for this target
-    #[error("unable to run linkage report for this type of binary; file was: {file_name:?}")]
-    LinkageCheckUnsupportedBinary {
-        /// The object's file name
-        file_name: Option<String>,
-    },
+    #[error("unable to run linkage report for this type of binary")]
+    LinkageCheckUnsupportedBinary,
```

Comment:
i do think maintaining the name of the file would be useful

Reply:
The warning all of these flow into handles reporting the filename so it ends up being redundant noise